### PR TITLE
[MME] Add libprotobuf-mutator to dockerfile for test

### PR DESCRIPTION
**Pull request by electronjoe**
## Summary

This library will be useful in writing protobuf based property testing (e.g. #6903, #5042, #4996) and possibly in fuzzers that validate healthy behavior on random protobufs.

## Test Plan

Confirmed health of Docker builds (via GH Action that checks this already).
Confirmed ability to add new unit test using the library (see draft PR #6892).

Signed-off-by: Scott Harrison Moeller <smoeller@fb.com>

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -153,6 +153,17 @@ RUN git clone --recurse-submodules -b v1.35.0 https://github.com/grpc/grpc && \
     cd / && \
     rm -rf grpc
 
+##### libprotobuf-mutator is used for randomized proto unit tests / property tests
+RUN git clone -b v1.0 https://github.com/google/libprotobuf-mutator && \
+    cd libprotobuf-mutator && \
```
Comment (tmdzk) [inline .devcontainer/Dockerfile]: you can mkdir directly into `libprotobuf-mutator` + cd into it. Save you one line

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -153,6 +153,17 @@ RUN git clone --recurse-submodules -b v1.35.0 https://github.com/grpc/grpc && \
...
+    mkdir -p build && \
+    cd build && \
+    cmake .. -GNinja -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_BUILD_TYPE=Debug && \
+    ninja && \
+    ninja install && \
+    cd / && \
```
Comment (tmdzk) [inline .devcontainer/Dockerfile:163]: Interesting haha

File: lte/gateway/docker/mme/Dockerfile.ubuntu20.04 [hunk 3]
```diff
@@ -134,6 +134,17 @@ RUN git clone --recurse-submodules -b v1.35.0 https://github.com/grpc/grpc && \
     cd / && \
     rm -rf grpc
 
+##### libprotobuf-mutator is used for randomized proto unit tests / property tests
+RUN git clone -b v1.0 https://github.com/google/libprotobuf-mutator && \
+    cd libprotobuf-mutator && \
+    mkdir -p build && \
```
Comment (tmdzk) [inline lte/gateway/docker/mme/Dockerfile.ubuntu20.04]: same

**Review by tmdzk — APPROVED** [review]

File: .devcontainer/Dockerfile (hunk 2 above)
Comment (electronjoe) [reply .devcontainer/Dockerfile:163]: Which part is interesting =D

Comment (electronjoe) [reply .devcontainer/Dockerfile:163]: Which par tis interesting =D

**Pull request merged**